controllers/function_controller.go

```diff
@@ -128,5 +139,6 @@ func (r *FunctionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.Service{}).
 		Owns(&autov2beta2.HorizontalPodAutoscaler{}).
 		Owns(&corev1.Secret{}).
+		Owns(&vpav1.VerticalPodAutoscaler{}).
```

Comment:
If the operator adds watches of `VerticalPodAutoscaler` to the controller, the function mesh operator will fail if the API doesn't exist in the K8s cluster. It might be better for the operator to check the existence of the API at the start time like this: https://github.com/streamnative/pulsar-operators/pull/669

Reply:
good point, I will update

Reply:
added